processing/src/test/java/org/apache/druid/segment/data/V3CompressedVSizeColumnarMultiIntsSerializerTest.java

```diff
@@ -322,7 +322,7 @@ private void checkV2SerializedSizeAndData(int offsetChunkFactor, int valueChunkF
           Assert.assertEquals(subVals.get(j), vals.get(i)[j]);
         }
       }
-      CloseQuietly.close(columnarMultiInts);
+      CloseableUtils.closeAndWrapExceptions(columnarMultiInts);
```

Comment:
Hmm, I know this is a test and probably doesn't really matter, but it seems like `mapper.close` should get called probably. Should this be using `closeAndSuppressExceptions` to ensure that it gets closed, or make them both part of a closer that gets passed into this method?

Reply:
I put them both in a single `CloseableUtils.closeAll`